Warehouse.Common/Extensions/EnumerableExtension.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Warehouse.Domain.Entities;
+
+namespace Warehouse.Common.Extensions
+{
+	public static class EnumerableExtension
```

Comment:
This is an extension only under IEnumerable of Products
This does not apply to all Enumerables cuz is not generic. The name should be changed

Reply:
Renamed to EnumerableProductExtension.